source/adapters/cuda/common.cpp

```diff
@@ -37,29 +38,34 @@ ur_result_t mapErrorUR(CUresult Result) {
 
 void checkErrorUR(CUresult Result, const char *Function, int Line,
                   const char *File) {
+  static bool suppress_error =
+      std::getenv("SYCL_PI_SUPPRESS_ERROR_MESSAGE") == nullptr &&
+      std::getenv("UR_SUPPRESS_ERROR_MESSAGE") == nullptr;
+
+  static bool abort = std::getenv("PI_CUDA_ABORT") != nullptr ||
+                      std::getenv("UR_CUDA_ABORT") != nullptr;
+
   if (Result == CUDA_SUCCESS || Result == CUDA_ERROR_DEINITIALIZED) {
     return;
   }
 
-  if (std::getenv("SYCL_PI_SUPPRESS_ERROR_MESSAGE") == nullptr &&
-      std::getenv("UR_SUPPRESS_ERROR_MESSAGE") == nullptr) {
+  if (suppress_error) {
```

Comment:
It looks like if `suppress_error` is true it doesn't suppress the error, right?

Shouldn't it be then named differently or defined differently/used differently?